composer.json

```diff
@@ -37,23 +36,6 @@
     "DDD",
     "PHP"
   ],
-  "authors": [
```

Comment:
why no authors any more? please leave that and also add the new symfony maintainers